### DISCUSSION TITLE
is it possible to have a button but which is declared with <a> tag with href for seo

### DISCUSSION THREAD

**Discussion by AkliOnet** [quasarframework/quasar#15827] [General - Components / Directives / etc] [answered]
based on google help page https://developers.google.com/search/docs/crawling-indexing/javascript/javascript-seo-basics#0
google can follow only a tag . so the question may be stupid but , are there a possibility to have the button component with the css that come with and an href ?  or what technique is advised to be sur that google index all our pages ?

**Comment by rstoenescu** [comment, marked as answer]
```html
<q-btn type="a" to="/my/route" />
<q-btn href="http://some.url" />
<q-btn href="http://some.url" target="_blank" />
```

When in doubt, it's always good to check the API cards on docs for the component in question.

**Reply by 3m1n3nc3** [reply]
To add to this, I think quasar will automatically generate an `<a>` tag if there is a link, that is whenever the `<q-btn>` component action is not handled by an @event handler.
That would include: `<q-btn :to="{ name: 'some.route' }"/>`
